# PointCloudWithCamera.msg
std_msgs/Header header
sensor_msgs/PointCloud2 point_cloud
VLM_grasper/Camera camera
